metricshub-engine/src/test/java/org/sentrysoftware/metricshub/engine/strategy/collect/ProtocolHealthCheckStrategyTest.java

```diff
@@ -534,4 +562,59 @@ void testCheckIpmiDownHealth() {
 			assertEquals(DOWN, telemetryManager.getEndpointHostMonitor().getMetric(IPMI_UP_METRIC).getValue());
 		}
 	}
+
+	@Test
+	void testCheckWbemUpHealthNamespaces() throws ClientException {
+		// Create a telemetry manager using a WBEM HostConfiguration
+		final TelemetryManager telemetryManager = createTelemetryManagerWithWbemConfig();
+
+		// Create a new protocol health check strategy
+		final ProtocolHealthCheckStrategy wbemHealthCheckStrategy = new ProtocolHealthCheckStrategy(
+			telemetryManager,
+			CURRENT_TIME_MILLIS,
+			clientsExecutorMock
+		);
+
+		// Mock a positive result for each WBEM test namespace

```

Comment:
This part can be changed to test various scenario of the WBEM health check method
```java
		{
			// Mock a positive response for every WBEM protocol health check test namespace
			doReturn(WQL_SUCCESS_RESPONSE)
				.when(clientsExecutorMock)
				.executeWbem(anyString(), any(WbemConfiguration.class), eq(WBEM_TEST_QUERY), anyString());

			// Start the WBEM Health Check strategy
			wbemHealthCheckStrategy.run();

			assertEquals(UP, telemetryManager.getEndpointHostMonitor().getMetric(WBEM_UP_METRIC).getValue());
		}

		{
			final Map<String, Answer<?>> answers = Map.of(
				"root/Interop", answer -> null,
				"interop", answer -> null,
				"root/PG_Interop",answer -> null,
				"PG_Interop", answer -> {
					throw new RuntimeException(new WBEMException(WBEMException.CIM_ERR_INVALID_NAMESPACE));
				}
			);

			for (Entry<String, Answer<?>> answerEntry : answers.entrySet()) {
				// Mock a positive response for every WBEM protocol health check test namespace
				doAnswer(answerEntry.getValue())
					.when(clientsExecutorMock)
					.executeWbem(anyString(), any(WbemConfiguration.class), eq(WBEM_TEST_QUERY), eq(answerEntry.getKey()));
			}

			// Start the WBEM Health Check strategy
			wbemHealthCheckStrategy.run();

			assertEquals(UP, telemetryManager.getEndpointHostMonitor().getMetric(WBEM_UP_METRIC).getValue());
		}

		{
			final Map<String, Answer<?>> answers = Map.of(
				"root/Interop", answer -> null,
				"interop", answer -> WQL_SUCCESS_RESPONSE
			);

			for (Entry<String, Answer<?>> answerEntry : answers.entrySet()) {
				// Mock a positive response for every WBEM protocol health check test namespace
				doAnswer(answerEntry.getValue())
					.when(clientsExecutorMock)
					.executeWbem(anyString(), any(WbemConfiguration.class), eq(WBEM_TEST_QUERY), eq(answerEntry.getKey()));
			}

			// Start the WBEM Health Check strategy
			wbemHealthCheckStrategy.run();

			assertEquals(UP, telemetryManager.getEndpointHostMonitor().getMetric(WBEM_UP_METRIC).getValue());
		}
		
		
```